src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line at the beginning of the class definition should be removed.